api/client/info.go

```diff
@@ -104,5 +104,16 @@ func (cli *DockerCli) CmdInfo(args ...string) error {
 		fmt.Fprintf(cli.out, "Experimental: true\n")
 	}
 
+	fmt.Fprintf(cli.out, "Docker Version Infomation: \n")
```

Comment:
Hi,

2 requests are necessary if you want global information on Docker.
For example in a monitoring script two requests can raise the execution time.
Moreover there are already some duplicate information as the kernel version.

Regards.